cmd/minikube/cmd/service.go

```diff
@@ -47,6 +55,26 @@ var serviceCmd = &cobra.Command{
 
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
+
+		s, err := cluster.GetHostStatus(api)
```

Comment:
Could you refactor this out? It looks basically the same as the dashboard.go implementation.